### DISCUSSION TITLE
[Projects Beta] Functionality to wrap text in Columns/Fields in GitHub projects.

### DISCUSSION THREAD

**Discussion by AbhilashAwasthi01** [community/community#8936] [Projects and Issues]
The fields in Github projects can be textual in nature sometimes. Currently there is no functionality to wrap text inside that field. Should be a functionality to wrap text inside a field.

**Comment by markgammon** [comment]
Or have a space above (similar to the existing search bar now) to view/edit the text in a column cell, such as in Excel or GSheet

**Comment by klonos** [comment]
+1 for this. It would be nice if for example the cells of the "Labels" column would wrap.
